opus/application/static_media/js/browse.js

```diff
@@ -663,13 +660,13 @@ var o_browse = {
     updateSliderHandle: function() {
         let tab = `#${opus.prefs.view}`;
         let contentsView = o_browse.getScrollContainerClass();
-        let selector = (opus.prefs.browse === "dataTable") ? `#${opus.prefs.view} #dataTable tbody tr` : `#${opus.prefs.view} .gallery .thumbnail-container`;
+        let selector = (opus.prefs[browse] === "gallery") ?  `#${opus.prefs.view} .gallery .thumbnail-container` : `#${opus.prefs.view} .op-data-table tbody tr`;
```

Comment:
I don't see how this can work. Isn't `browse` an undefined variable here? Maybe this is why the slider isn't working some of the time.